evmjit/libevmjit/Arith256.cpp

```diff
@@ -3,9 +3,12 @@
 #include "Type.h"
 #include "Endianness.h"
 
-#include <llvm/IR/Function.h>
+#include "preprocessor/llvm_includes_start.h"
 #include <llvm/IR/IntrinsicInst.h>
+#include "preprocessor/llvm_includes_end.h"
+
 #include <iostream>
```

Comment:
order of includes. General libs such as iostream should be in front of special/local libs. (C, STL, Boost, other libs, base ethereum libs, current ethereum lib)